contribution_to_project.rst

```diff
@@ -0,0 +1,95 @@
+***********************
+Contributing to Ansible
+***********************
+
+This document describes ways how you can contribute to the Open Source projects of the Ansible ecosystem.
+
+.. contents:: Topics
+
+What is a contribution to an Open Source project
+================================================
+
+A contribution to an Open Source project is any good given by an individual or a company to the project including increasing its functionality, improvements of the codebase, code developer experience, user experience, increasing the project's popularity and community, providing infrastructure, and any other kinds of active social or material support.
+
+Most types of contribution to an Open Source project do not require programming skills.
+
+Classification of contributions
+===============================
+
+Contributions can be one of the following types:
+
+  - :ref:`Contribution to a codebase<Contribution to codebase>`.
```

Comment:
This list repeats the items from the paragraph above. I'd remove the paragraph and the extra heading, and head this list "Some ideas for contributing to an open-source project:"

Reply:
Removed, thanks